pages/index.tsx

```diff
@@ -112,23 +146,99 @@ const Home: NextPage = () => {
               }
             />
           </div>
-          <button className="w-max p-2.5 px-4 text-gray-600">
-            <FontAwesomeIcon icon={faRedo} />
-          </button>
-        </div>
-        <div className="container mx-auto grid sm:grid-cols-2 lg:grid-cols-3 gap-6">
-          {data?.slice(0, 20).map((font) => (
-            <FontCard
-              key={font.family}
-              data={font}
-              fontSize={fontSize}
-              onClick={(font) => {
-                push(`/speciment/${font.family}`)
+          <div className="p-2 flex flex-col justify-center">
+            <Button
+              icon="redo"
+              onClick={() => {
+                setKeyWord('')
+                setFontSize(40)
+                setPreviewText(
+                  'Almost before we knew it, we had left the ground.'
+                )
+                const previewInputVal = document.getElementById(
+                  'previewInput'
+                ) as HTMLInputElement
+                if (previewInputVal !== null) previewInputVal.value = ''
+                const searchInputVal = document.getElementById(
+                  'searchInput'
+                ) as HTMLInputElement
+                if (searchInputVal !== null) searchInputVal.value = ''
               }}
+              className="h-8 w-8 grid items-center justify-center text-xl rounded-full bg-white text-gray-600 disabled:text-gray-400 hover:bg-gray-50 hover:text-gray-800"
+              disabled={
+                previewText !==
+                  'Almost before we knew it, we had left the ground.' ||
+                fontSize !== 40 ||
+                keyWord !== ''
+                  ? false
+                  : true
+              }
```

Comment:
This can be simplified as:
```suggestion
              disabled={
                previewText !==
                  'Almost before we knew it, we had left the ground.' ||
                fontSize !== 40 ||
                keyWord === ''
              }
```

However, I would not recommend a long chain of logical OR like that